### DISCUSSION TITLE
Documenting "Missing" Features

### DISCUSSION THREAD

**Discussion by askreet** [helix-editor/helix#6367] [Ideas] [answered]
In the discussions here, on reddit and in Matrix we often see people asking the same questions about certain categories of features that are "missing". In some cases they're truly not implemented yet and in other cases the maintainers have decided against implementing them.

Often the (very helpful!) community members paste links to the same issues where progress is being tracked and workarounds are offered by various folks. A recent example is someone pointing out the lack of integrated terminal offering a way to use Kitty to send a command to another window pane.

I wonder if a documentation page on helix-editor.com titled something like `"Missing" Features` that has pages that:

* Are titled for the missing feature e.g. `Terminal Mode`
* Contain links to any relevant discussion, issues and work-in-progress PRs for the feature.
* Contain any community suggested workarounds or alternatives in a central place.

In theory this would:
* Lower the burden of answering these questions ("See [link]...")
* Increase the default quality of the answers to the sum of all available known workarounds, alternatives, etc.

The downside would be keeping this documentation up to date with the rapid pace of development on these features.

I'd be willing to pick a couple features and scaffold out this documentation section if people think this is a good idea.

**Comment by pascalkuthe** [comment, marked as answer]
I am not sure about this because it implies that these features are accepted and just "missing" an implementation. People ask for a lot of features in issue but that doesn't mean they get implemented. If there is some official documentation for a feature people will assume it would definitely be added in the future (so questions would turn from "is this implemented/planned" to "any progress on X?" or "when is X expected to land" and similar). 

Adding stuff to the FAQ might be ok tough if it doesn't imply anything commital like:
 
> Does helix have an integrated terminal? 
>
> Not currently, see <ISSUE> for discussion, a rough prototype was published in <PR>

**Reply by askreet** [reply]
I actually explicitly want to include things which are _not_ accepted and state that we don't plan on including them, too. I like your suggestion for including them in the FAQ. I actually hadn't seen the FAQ!

Any thoughts on whether the FAQ should be cloned or moved into helix-editor.com? Anecdotally, I would think people are more likely to read the .com's documentation section than travel to GitHub (even with the link), but I have no data to back that up :-).
